### PR TITLE
Update skope_rules.py (fix invalid mask indexing for OOB samples)

### LINKED ISSUE #38: issue in mask indexing
Hi, thank you for sharing this great package.

However, I think I might find a mistake in the mask indexing.

`mask = ~samples`

samples is numpy array, and when you put ~, you can get -(value+1).

ex.
samples = np.array([1,2,3,4])
~samples
[-2, -3, -4, -5]

please check this issue.

Thanks!

### PR DESCRIPTION
**Pull request by stat17-hb**
fix #38 
close #47 
please check the code for mask indexing.

### CODE REVIEW

File: skrules/skope_rules.py [hunk 1]
```diff
@@ -334,7 +334,10 @@ def fit(self, X, y, sample_weight=None):
                                                 self.estimators_features_):
 
             # Create mask for OOB samples
-            mask = ~samples
+            # mask = ~samples # samples is numpy array. when you put ~. it returns array([-(value_1 + 1), ..., -(value_n + 1)])
+            mask = list(set(np.arange(len(samples))).difference(samples))
```
Comment (ngoix) [inline skrules/skope_rules.py:338]: could you do something like `from sklearn.utils import indices_to_mask` and `mask = ~indices_to_mask(samples, n_samples)`?

**Comment by ngoix** [comment]
thanks for opening this PR

File: skrules/skope_rules.py (hunk 1 above)
Comment (stat17-hb) [reply skrules/skope_rules.py]: > could you do something like `from sklearn.utils import indices_to_mask` and `mask = ~indices_to_mask(samples, n_samples)`?

Thank you for your kind comment!
By the way, when I did commit again, It still got errors in travis CI(29.3 - python 3.5 environment...)

File: skrules/skope_rules.py [hunk 2]
```diff
@@ -334,7 +335,9 @@ def fit(self, X, y, sample_weight=None):
                                                 self.estimators_features_):
 
             # Create mask for OOB samples
-            mask = ~samples
+            # mask = ~samples # samples is numpy array. when you put ~. it returns array([-(value_1 + 1), ..., -(value_n + 1)])
```
Comment (ngoix) [inline skrules/skope_rules.py:338]:
```suggestion

```
